### DISCUSSION TITLE
FastAPI authentication with JWT Token not working in browser tab but works in Swagger UI

### DISCUSSION THREAD

**Discussion by csharmatech** [fastapi/fastapi#5827] [Questions]
Hello,

I am exploring FastAPI authentication with JWT as described in [https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/](url) . I am using MYSQL as backend DB for authenticating users. I am able to authenticate a user using custom HTML login form and once authenticated, the user gets access token and it prints on the page. Now, if I open a new tab and goto path "/users/me" for verifying current user as per docs [https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/](url) , I get message as "Not authenticated". This works perfectly fine in the Swagger UI. 

I am not able to understand this?

**Comment by caeser1996** [comment]
When you authenticate a user and get an access token, it is stored in the user's browser as a cookie or local storage. When you open a new tab and access the "/users/me" endpoint, the access token is not being sent to the server. This is why you are getting the "Not authenticated" message.

To resolve this issue, you need to make sure that the access token is being sent along with each request to the server, either as a cookie or as an Authorization header. The exact method will depend on the implementation details of your application and how you are storing the access token.

If you are using cookies, you will need to make sure that the browser is sending the cookie with each request. If you are using an Authorization header, you will need to make sure that it is being set for each request. You can use tools like the developer tools in your web browser to inspect the request headers and verify that the access token is being sent correctly.
